### ISSUE TITLE
Two ships dock, only one can move

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#5742]
### Short Description of the issue:
When two ships are docked, the one that initiated the docking cannot move. When you fly it, it remains fixed in space although the dynamic camera moves

https://imgur.com/qvMzGHx

### Steps to reproduce the issue:
- Dock two of the attached bps
- Fly the ship that you hit the switch on

### Environment
- Starship EVO version: 24w18a
- Graphics Card: 

### Additional informatio
[Docking Test Dummy.zip](https://github.com/tsunamayo/Starship-EVO/files/15210969/Docking.Test.Dummy.zip)
n:

**Comment by TIKIRobo** [comment]
I feel like if the system requires one entity to be inactive and one to be active it could be determined by mass or core type? for example a station (an entity w/ a station core on it) would always be the active one so that the docked ships can't take it away or for mass based the smaller mass/sized entity would be inactive so a carrier would be able to move even if the fighter initiates docking 

also if it doesn't do this already the smaller mass should move towards the larger one when docking

**Comment by Kadzar** [comment]
If the larger mass is always active then it removes the possibility of tugships that bring a larger ship into port or cargo ships that transport large removeable cargo modules.

It think it probably would be better to have a system where ships do or do not cede movement control while docked, preferably toggleable with logic. That way you can decide on a per-situation basis what's appropriate.

**Comment by tsunamayo** [comment]
Yes I want to add power and other system stats sharing (thrust ect), but it will take some time it is not a small dev. Cheers
